tests/testthat/tests_general.R

```diff
@@ -296,56 +296,57 @@ test_that("readWQPdots working", {
 context("NGWMN")
 test_that("NGWMN functions working", {
   testthat::skip_on_cran()
-  noDataSite <- "UTGS.401544112060301"
-  noDataSite <- readNGWMNlevels(siteNumbers = noDataSite)
-  expect_true(is.data.frame(noDataSite))
-  
-  #bounding box and a bigger request
-  bboxSites <- readNGWMNdata(service = "featureOfInterest", bbox = c(30, -99, 31, 102))
-  expect_gt(nrow(bboxSites), 0)
+  # noDataSite <- "UTGS.401544112060301"
+  # noDataSite <- readNGWMNlevels(siteNumbers = noDataSite)
+  # expect_true(is.data.frame(noDataSite))
+  # 
+  # #bounding box and a bigger request
+  # bboxSites <- readNGWMNdata(service = "featureOfInterest", bbox = c(30, -99, 31, 102))
+  # expect_gt(nrow(bboxSites), 0)
   # siteInfo <- readNGWMNsites(bboxSites$site[1:3])
   # expect_equal(nrow(siteInfo), 3)
   
   #one site
-  site <- "USGS.430427089284901"
```

Comment:
In the event that the NGWMN slowdown is a parsing issue, can we have a test that fails after a certain time if something doesn't complete?

Reply:
yeah, that sounds good